modules/audio/mic.py

```
@@ -0,0 +1,96 @@
+                    stream = AkariDataStreamType(frames)
+                    dataset.audio = AkariDataSetType(main=b"".join(frames), stream=stream)
+                    data.add(dataset)
+                    if callback is not None:
+
+                        def call_module_in_thread() -> None:
+                            nonlocal abc
+                            print(abc)
+                            abc += 1
+                            self._router.callModule(
+                                moduleType=callback,
+                                data=data,
```

Ruff (0.8.2)

Function definition does not bind loop variable `data`

(B023)